_posts/2022-04-01-this-year-in-monero.md

```diff
@@ -0,0 +1,223 @@
+
+### Year over Year Growth
+
+Monero's network hashrate increased ***53.27%*** YoY
+
+## Reddit
+
+***04/2020 through 04/2021:***
+
+- Monero's main subreddit, [r/Monero](https://www.reddit.com/r/Monero) gained ***~43,000*** subscribers, going from ***~168,000*** to ***~211,000*** subscribers over the year.
+
+<img src="/blog/assets/2021-04-18-this-year-in-monero/subreddit_2021.png" position="center" style="border-radius: 8px;">
+
+***04/2021 through 04/2022:***
+
+- Monero's main subreddit, [r/Monero](https://www.reddit.com/r/Monero) gained ***~46,000*** subscribers, going from ***~211,000*** to ***257,000*** subscribers over the year.
+
+<img src="/blog/assets/2022-04-01-this-year-in-monero/subreddit_2022.png" position="center" style="border-radius: 8px;">
+
+### Year over Year Growth
+
+Monero's subreddit subscriber count grew ***21.80%*** YoY
+
+### Top Ten Reddit Posts
+
+- [I'm glad Monero is used by criminals](https://www.reddit.com/r/Monero/comments/mpf8b3/im_glad_monero_is_used_by_criminals/)
+- [I don't need Monero](https://www.reddit.com/r/Monero/comments/n5br07/i_dont_need_monero/)
+- [Probably a Repost but still very funny](https://www.reddit.com/r/Monero/comments/ocvi5x/probably_a_repost_but_still_very_funny/)
+- [John McAfee supporting privacy coins](https://www.reddit.com/r/Monero/comments/o6l8rb/john_mcafee_supporting_privacy_coins/)
+- [I'm a huge advocate for privacy and this just makes me sad](https://www.reddit.com/r/Monero/comments/pd9jnm/im_a_huge_advocate_for_privacy_and_this_just/)
+- [Five minutes of fame or ...](https://www.reddit.com/r/Monero/comments/nzro3v/five_minutes_of_fame_or/)
+- [Hal Finney, the first person to receive Bitcoin from Satoshi, took 11 days to realized he wanted more anonymity.](https://www.reddit.com/r/Monero/comments/mo1m2a/hal_finney_the_first_person_to_receive_bitcoin/)
+- [Haveno's CCS proposal is now open for donations!](https://www.reddit.com/r/Monero/comments/tsvdwl/havenos_ccs_proposal_is_now_open_for_donations/)
+- [Happy birthday, Monero! Here's to another 7 years!](https://www.reddit.com/r/Monero/comments/msrbp4/happy_birthday_monero_heres_to_another_7_years/)
+- [Official Report - What Monero Users Spend Their Crypto On](https://www.reddit.com/r/Monero/comments/n2yviz/official_report_what_monero_users_spend_their/)
+
+## Twitter
+
+***04/2020 through 04/2021:***
+
+- Monero's main Twitter account, [@monero](https://twitter.com/monero) gained ***69,209*** followers.
+- The @monero Twitter account went from ***319,884*** to ***389,093*** followers.
+
+***04/2021 through 04/2022:***
+
+- Monero's main Twitter account, [@monero](https://twitter.com/monero) gained ***84,397*** followers.
+- The @monero Twitter account went from ***389,093*** to ***473,490*** followers.
+
+### Year over Year Growth
+
+Monero's Twitter follower count grew ***21.69%*** YoY
+
+### Recent Popular Tweets
```

Comment:
I would remove this section as well

Reply:
Remove the top tweets section entirely?

Reply:
Yeah in hind-sight I agree. Cleaner without and some of the posts aren't... fitting for the official blog.